polish-adblock-filters/adblock.txt

```diff
@@ -5532,6 +5532,10 @@ poczta.wp.pl##.ContainerBox native-feed[ng-if^="ListDirectiveController.currentL
 poczta.wp.pl##.sidebar__outline > sidebar-feed
 poczta.wp.pl##[slot^=BOX_LEFT_]
 poczta.wp.pl##[slot=SKY]
+poczta.wp.pl##.css-1y78uuz
+poczta.wp.pl##html:style(position: static !important; overflow: visible !important)
+poczta.wp.pl##body:style(position: static !important; overflow: visible !important)
+poczta.wp.pl##:not(.overflow-x_hidden, .overflow_hidden, .truncate_true, svg:not(:root), img):style(overflow: visible !important)
```

Comment:
Linie z CSS-em wklejone do złego pliku, te są poprawne:

https://github.com/MajkiIT/polish-ads-filter/blob/master/polish-adblock-filters/adblock_ublock.txt
https://github.com/MajkiIT/polish-ads-filter/blob/master/polish-adblock-filters/adblock_adguard.txt

Reply:
Dzięki za szybki przegląd.
Przeniosłem rulesy z CSSem do adblock_ublock.txt. 
`.css-1y78uu` to klasa którą znalazłem tylko w jednym divie (nad listą wiadomości) w którym wyświetlane są reklamy. 

Jeżeli chodzi o ABP i AdBlock to nie wiem jak to wygląda bo używam tylko ublocka. W wolnej chwili mogę do tego usiąść, ale bez modyfikacji styli a jedynie przy filtrowaniu elementów będzie ciężko raczej bo `overflow: hidden; position: fixed;` jest dodany do taga `<html>`.

Reply:
Jest cień szansy, że tam (ABP/AdBlock) nie ma awarii i robi ją jakiś filtr do uBo.

W przypadku kosmetycznych filtrów ukrywających można zastosować pomysł śp. gwarsera:

```adb
! w pliku dla większości blokerów treści
example.com#@#.ads
! w pliku _ublock.txt
example.com##.ads.ads
```

Teoretycznie też działają if-y: https://github.com/gorhill/ublock/wiki/static-filter-syntax#if-condition


```adb
!#if ext_abp
example.com#@#.ads
!#endif
```

Reply:
Sprawdziłem przed chwilą na chromie z AdBlockiem i też nie jest ukrywany ten element: `.css-1y78uu`.
![Zrzut ekranu 2025-07-09 002804](https://github.com/user-attachments/assets/e3afcb6d-7fc0-4aa2-b0bd-730176bc5950)

Nie jestem pewny jaka logika za tym stoi, ale nie ma tego elementu na wszystkich podstronach. Np. w Odebrane/Główne ten element nie jest obecny, a w spamie już jest.